metpy/calc/thermo.py

```diff
@@ -103,9 +103,9 @@ def potential_temperature(pressure, temperature):
     Parameters
     ----------
     pressure : `pint.Quantity`
-        The total atmospheric pressure
+        total atmospheric pressure
     temperature : `pint.Quantity`
-        The temperature
+        temperature
```

Comment:
This seems silly. Is there no better description we can put here?

Reply:
Going with `air temperature` as that was already used throughout.